about.md

```diff
@@ -40,7 +96,8 @@ Unlike a traditional CMS, Pico doesn't require write access on your server to fu
 
 There really isn't that much more to say about it.  The beauty of Pico is in its simplicity.  If you want to create a new page, you make a new file.  That's it.  But that's far from the end of the story.  Despite Pico's simplicity, you'll find it incredibly powerful, and ready to take on any task.
 
-{% include fancyboxtemplate.html, gallery: workflow %}
```

Comment:
btw: The Jekyll docs say that it's `{% include footer.html param="value" variable-param=page.variable %}`. However, this has never worked for me, too :wink: Maybe it works now since GitHub Pages switched to Jekyll 3 recently.